aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -3598,4 +3611,31 @@ public String toString()
             "election=" + election +
             '}';
     }
+
+    private static final class UnknownConsensusModuleExtension implements ConsensusModuleExtension
+    {
+
+        private static final UnknownConsensusModuleExtension INSTANCE = new UnknownConsensusModuleExtension();
+        private UnknownConsensusModuleExtension()
+        {
+        }
+
+        @Override
```

Comment:
We don't use @Override in our code style.

Reply:
removed